vscode-lean4/test/suite/utils/helpers.ts

```diff
@@ -4,6 +4,87 @@ import { privateEncrypt } from 'crypto';
 import * as vscode from 'vscode';
 import { InfoProvider } from '../../../src/infoview';
 import { LeanClient} from '../../../src/leanclient';
+import * as ps from 'ps-node';
+
+export async function findProcs(name: string) : Promise<ps.Program[]> {
+  // A simple pid lookup
+  return await new Promise<ps.Program[]>((resolve) => {
+    ps.lookup({ command: name }, function(err, resultList ) {
+      if (err) {
+        resolve([]);
+      }
+      resolve(resultList);
+    });
+  });
+}
+
+// find out how many lean --server, and lean --worker processes are running
+// and return both counts.
+export async function findLeanServers() : Promise<[number,number]>{
+    // BUGBUG: this is not working reliably... we seem to be leaking --worker processes
+    // sometimes, especially in the Orphan files test.
```

Comment:
I wouldn't be surprised if it takes a second for the server and worker processes to terminate.

There is another issue with this function: it looks at all running processes, and not just the ones which are children of the test suite.  So you will get bogus results if you have a lean editor open in the background.

Reply:
Yes, I've punted this to https://github.com/leanprover/vscode-lean4/issues/144 to figure out a better solution.

Reply:
All right.  Could you please remove the process stuff for now and change the PR title?  Otherwise the PR looks fine.

Reply:
Done.